src/components/search/entry/SortSelect.jsx

```diff
@@ -71,22 +71,31 @@ class SortSelect extends Component {
     } = this.props;
 
     return (
-      <select
-        className={styles.common}
-        value={value}
-        onChange={this.handleChange}
-      >
-        {
-          ['bestmatch', 'atoz', 'ztoa', 'newest', 'oldest'].map((sortOrder) => (
-            <option
-              key={sortOrder}
-              value={sortOrder}
-            >
-              {intl.formatMessage(messages[sortOrder])}
-            </option>
-          ))
-        }
-      </select>
+      <>
+        <label
+          className={styles.commonLabel}
+          aria-role="region"
+          aria-label="Sort order"
+        >
```

Comment:
There is nothing here that actually associates this `label` with the `select`. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label.

My preference is to enclose the `select` inside the `label` instead of using the `id` and `for` attributes. Then you won't need the React fragment, and you can just return the `label`.

I don't think the `aria-role` and `aria-label` attributes are needed. Were they asked for, specifically? The role of a `label` is to already known to be a label, so they seem redundant. See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label.

Reply:
Fixed